src/Symfony/Component/Validator/Constraints/BicValidator.php

```diff
@@ -68,11 +69,12 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
-        // @deprecated since Symfony 4.2
+        // @deprecated since Symfony 4.2, will throw in 5.0
         if (class_exists(Intl::class)) {
             $validCountryCode = isset(Intl::getRegionBundle()->getCountryNames()[substr($canonicalize, 4, 2)]);
         } else {
             $validCountryCode = ctype_alpha(substr($canonicalize, 4, 2));
+            // throw new LogicException('The "symfony/intl" component is required to use the Bic constraint.');
```

Comment:
We should trigger a deprecation here for now (at least when the passed constraint is a subclass of the `Bic` constraint).

Reply:
agreed, sounds like a very edge case afterall

Reply:
fair enough, but I think we should still throw the exception in 5.0